packages/docs/pages/users/governance/proposal-types.mdx

```diff
@@ -0,0 +1,92 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Types of governance proposals
+
+There are three types of governance proposals: default, PGF, and Steward. Below, the differences are described by showing what the `data` field in the governance proposal typically looks like.
+
+## Default Proposal 
```

Comment:
should we explain somewhere the different quorums needed to pass each proposal type or leave it for the specs?

Reply:
Thx for pointing this out, yes I agree we should. Will do this soon after sorting out a small possible gov bug related to one of these different quora.

Reply:
done in 4910698f864013c1815c35161b0191adcee7c289